src/Core/src/Handlers/Image/ImageHandler.Windows.cs

```diff
@@ -8,33 +9,111 @@ namespace Microsoft.Maui.Handlers
 {
 	public partial class ImageHandler : ViewHandler<IImage, Image>
 	{
+		/// <inheritdoc/>
 		protected override Image CreatePlatformView() => new Image();
 
+		/// <inheritdoc/>
 		protected override void DisconnectHandler(Image platformView)
 		{
 			base.DisconnectHandler(platformView);
 			SourceLoader.Reset();
 		}
 
-		public override bool NeedsContainer =>
-			VirtualView?.Background != null ||
-			base.NeedsContainer;
+		/// <inheritdoc/>
+		public override bool NeedsContainer => true;
```

Comment:
This doesn't need the container if the Aspect isn't AspectFill (and there's no background set).